.github/workflows/continuous-integration-workflow.yml

```diff
@@ -19,3 +19,48 @@ jobs:
 
     - name: Run Docker
       run: docker run -e FLASK_APP=${{ secrets.FLASK_APP_X}} -e FLASK_ENV=${{ secrets.FLASK_ENV_X}} -e SECRET_KEY=${{ secrets.SECRET_KEY_X }} todo-app:devtest
+
+  job-two:
+    name: image build & push 
+    runs-on: ubuntu-latest
+    needs: job-one
+    steps:
+      - 
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
I know this is recommended by the build-push-action but if we look at what it's actually for it's not relevant for us; [it's for adding emulation support so we can build for additional platforms](https://github.com/marketplace/actions/build-and-push-docker-images#usage), which we won't need to do